app/controllers/income/estimatedPay/update/IncomeUpdateCalculatorController.scala

```diff
@@ -430,7 +428,8 @@ class IncomeUpdateCalculatorController @Inject()(incomeService: IncomeService,
 
                 val payPeriod = optionalSeq(0)
                 val payPeriodInDays = optionalSeq(1)
-                TaxablePaySlipAmountViewModel(TaxablePayslipForm.createForm(), payPeriod, payPeriodInDays, id, employerName)
+                val errorMessage = "tai.taxablePayslip.error.form.incomes.radioButton.mandatory"
```

Comment:
There is an alternative way of doing this that I have put on branch ST-459-alt. I think it is best to go with that. Happy to discuss.